### ISSUE TITLE
Right windows button lights blue all the time (SteelSeries Apex M750 TKL)

### ISSUE THREAD

**Issue by fraannk** [antonpup/Aurora#1306]
### Expected behavior:
Right windows button should follow whatever lighting mode Aurora is in. (e.g. Overwatch)

### Actual behavior:
All other keys than the right windows button works great. The right windows button lights up persistent in a blue color. 

### How often does this problem occur?
All the time, no matter the effects used. Works fine when using only SteelSeries Engine 3

### Please describe the steps to reproduce this issue:
The issue can be reproduced by doing following steps:

1. Get an Apex M750 TKL
2. Install Engine 3 and Aurora and follow setup guides. 
3. Disable all effects other than Overwatch.
4. Open Overwatch and see the right windows button light up persistently, no matter the hero chosen. 

### What is the version of Aurora you are experiencing this issue with?
Beta 0.6.3

Note: I am using the Nordic layout of the Apex M750 TKL.

**Comment by simon-wh** [comment]
It appears that the Apex M750 TKL layout is slightly off and doesn't have the right windows key, resulting in it not being changed from the default. I assume it happens no matter what you're using to light it up? i.e. using solid fill layer on the desktop?

**Comment by fraannk** [comment]
Correct. The right windows button never turns off, always a baby blue color.
